requirements.txt

```diff
@@ -8,3 +8,4 @@ xarray
 zarr
 fsspec
 s3fs
+xarray-datatree==0.0.4
```

Comment:
Is this equality because this is being actively developed?

Reply:
Yepp, it will be developed heavily, so I wanna make sure nothing breaks on our end unexpectedly.

Reply:
Makes sense!